### PR TITLE
VNETpeeringExamples

### PR DESCRIPTION
**Pull request by anavinahar**
Updated help for VNetPeering cmdlets

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@anavinahar__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (annahar). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManager/Network/Commands.Network/Microsoft.Azure.Commands.Network.dll-Help.xml [hunk 1]
```diff
@@ -7178,6 +7177,68 @@ Subnets           : []</dev:code>
...
+Id                               : /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/MyResourceGroup/provi
+                                   ders/Microsoft.Network/virtualNetworks/MyVirtualNetwork/virtualNetworkPeerings/LinkToVNet2                     
+Etag                             : W/&quot;xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx&quot;
+ResourceGroupName                : MyResourceGroup
+VirtualNetworkName               : MyVirtualNetwork
+ProvisioningState                : Succeeded
+RemoteVirtualNetwork             : {
+                                     &quot;Id&quot;: &quot;/subscriptions/XXX/resourceGroups/MyResourceGroup
+                                   /providers/Microsoft.Network/virtualNetworks/MyVirtualNetwork&quot;
+                                   }
```
Comment (DeepakRajendranMsft) [inline src/ResourceManager/Network/Commands.Network/Microsoft.Azure.Commands.Network.dll-Help.xml]: I made a fix to output the peering state, could that property also be added in the example?

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

**Comment by markcowl** [comment]
@anavinahar Please respond to feedback from the service team

**Comment by anavinahar** [comment]
@DeepakRajendranMsft What changes did you make? I ran the example in PS again and didn't catch the change.

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

**Comment by anavinahar** [comment]
Updated Peering State as asked. https://github.com/Azure/azure-powershell/pull/2690/commits/f988a26fd62fc5664137cf58a57ac19852e8e7b3

**Comment by markcowl** [comment]
on demand run here: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1049/

**Comment by markcowl** [comment]
@deepakrajendranmsft can you sign off?

**Comment by DeepakRajendranMsft** [comment]
LGTM

**Comment by markcowl** [comment]
LGTM once the on-demand build passes

**Pull request merged**